src/governance/governance.cpp

```diff
@@ -771,9 +770,9 @@ void CGovernanceManager::VoteGovernanceTriggers(const std::optional<const CGover
 bool CGovernanceManager::VoteFundingTrigger(const uint256& nHash, const vote_outcome_enum_t outcome, CConnman& connman,
                                             const CActiveMasternodeManager* const mn_activeman)
 {
-    if (!fMasternodeMode) return false;
+    if (mn_activeman == nullptr) return false;
```

Comment:
drop check and replace to reference same as `CreateGovernanceTrigger`

Reply:
Resolved in latest push!